debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/launchConfigurations/LaunchConfigurationsDialog.java

```diff
@@ -1409,6 +1416,20 @@ public void run(boolean fork, boolean cancelable, IRunnableWithProgress runnable
 		}
 	}
 
+	private UIJob createJobToShowProgressMonitorPart() {
+		return new UIJob(IInternalDebugCoreConstants.EMPTY_STRING) {
```

Comment:
Besides UIJob.create() I think we can just use an empty string literal here.
IMHO it does not make sense to get that from a constant.

Reply:
Done in 6c443cccd3e7d5912b17855cfcdd8e5535e8663e